### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by kevinwilson541**

### CODE REVIEW

**Review by jaredallard — APPROVED** [review]

**Comment by jaredallard** [comment]
can we get this merged ASAP? thanks

**Comment by jaredallard** [comment]
@austinbrown-okta please get this merged

**Review by austinbrown-okta — APPROVED** [review]
All changes are for docs -- lgtm

File: README.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
 Clusterluck
```
Comment (ncthbrt) [inline README.md:1]: Should the project name in the readme not change as well?

**Pull request merged**